addons/web/static/src/core/install_prompt/install_prompt.scss

```diff
@@ -0,0 +1,30 @@
+.o_install_prompt {
+    border-radius: 1em;
+    height: unset !important;
+	-webkit-backdrop-filter: blur(5px);
+	backdrop-filter: blur(5px);
+	background: rgba(#FFF, .7);
+
+    &.o_touch_bounce {
+        animation: none;
+    }
+
+    &.mobile {
+        width: 90% !important;
+        inset: auto auto 5px 50% !important;
```

Comment:
Why "5px" ?

Reply:
So the dialog is not touching the borders of the screen. I guess I may instead use margins